client/src/features/signup/data/formModel.js

```diff
@@ -37,13 +37,13 @@ export default {
     invalidErrorMsg: 'Passwords must match',
   },
 
-  phone: {
-    name: 'phone',
+  telephone: {
+    name: 'telephone',
     label: 'Phone',
```

Comment:
Please, make this field as required. I can go ahead and not type any data at this field and then catch an err

![image](https://user-images.githubusercontent.com/112622259/204810763-aa3e31c8-83dc-40c0-a316-b89b0aee49a7.png)